packages/@pollyjs/ember/tests/index.html

```diff
@@ -1,33 +1,29 @@
 <!DOCTYPE html>
```

Comment:
Auto generated by ember-cli. This should be ignored from linting.

Reply:
Done.